types/validator.go

```diff
@@ -116,14 +116,10 @@ func ValidatorListString(vals []*Validator) string {
 // as its redundant with the pubkey. This also excludes ProposerPriority
 // which changes every round.
 func (v *Validator) Bytes() []byte {
-	pk, err := ce.PubKeyToProto(v.PubKey)
-	if err != nil {
-		panic(err)
-	}
-
 	pbv := cmtproto.SimpleValidator{
```

Comment:
Should we also check `v.PubKey` for `nil` here, as you do below in line 138?

Reply:
I don't see a point. If v.PubKey is nil, we'll panic with a similar message.